Source/Utils/WarpXAlgorithmSelection.cpp

```diff
@@ -0,0 +1,88 @@
+#include <WarpXAlgorithmSelection.H>
+
```

Comment:
I find that I need to `#include <map>` here to compile this on Summit.

Reply:
OK, thanks. I'll add this.